Don't append optimization flags -- rely on what the building
machine has set.
--- config/freebsd	2012-03-17 23:44:05.000000000 -0400
+++ config/freebsd	2012-04-21 10:30:43.000000000 -0400
@@ -102,5 +102,5 @@
     DEBUG_CFLAGS="-g -ansi -Wall -pedantic "
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-ansi -Wall -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wconversion -Wmissing-prototypes -Wnested-externs -pedantic -O2"
+    PROD_CFLAGS="-ansi -Wall -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wconversion -Wmissing-prototypes -Wnested-externs -pedantic"
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
@@ -112,5 +112,4 @@
     DEBUG_CFLAGS="-g"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O"
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
@@ -123,5 +122,4 @@
     FFLAGS="$FFLAGS"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
     PROFILE_FFLAGS="-pg"
     ;;
@@ -130,5 +128,4 @@
     FFLAGS="$FFLAGS"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
     PROFILE_FFLAGS="-pg"
     ;;
@@ -139,5 +136,4 @@
     CXXFLAGS="$CXXFLAGS"
     DEBUG_CXXFLAGS="-g -fverbose-asm"
-    PROD_CXXFLAGS="-O3 -fomit-frame-pointer"
     PROFILE_CXXFLAGS="-pg"
     ;;
@@ -146,5 +142,4 @@
     CXXFLAGS="$CXXFLAGS -ansi"
     DEBUG_CXXFLAGS="-g"
-    PROD_CXXFLAGS="-O"
     PROFILE_CXXFLAGS="-pg"
     ;;
